### ISSUE TITLE
Error: Setting unknown property: craft\\web\\twig\\variables\\Paginate::limit

### ISSUE THREAD

**Issue by slebbo** [markhuot/craftql#263]
After updating to the latest version of Craft, something what seems to be Twig-related stopped working. 
 
My CraftQL query:
```
{
  categoriesConnection(slug: "catty") {
    edges {
      node{
        title
      }
    }
  }
}
```

The response
```
{
  "error": "Setting unknown property: craft\\web\\twig\\variables\\Paginate::limit"
}
```

I only ran into the error when using those categoryConnection queries.

My setup:
Craft edition & version | Craft Solo 3.1.22
CraftQL | 1.3.1
Yii version | 2.0.17
Twig version | 2.7.4
Guzzle version | 6.3.3
PHP version | 7.3.1
OS version | Linux 4.15.0-45-generic
Database driver & version | MySQL 5.5.5

**Comment by markhuot** [comment]
This should be fixed by the 1.3.2 release

**Comment by slebbo** [comment]
Wow! Thanks for that update! I just started to rewrite a workaround in Element API :D
